hcn/hcnloadbalancer.go

```diff
@@ -10,10 +10,11 @@ import (
 
 // LoadBalancerPortMapping is associated with HostComputeLoadBalancer
 type LoadBalancerPortMapping struct {
-	Protocol     uint32                       `json:",omitempty"` // EX: TCP = 6, UDP = 17
-	InternalPort uint16                       `json:",omitempty"`
-	ExternalPort uint16                       `json:",omitempty"`
-	Flags        LoadBalancerPortMappingFlags `json:",omitempty"`
+	Protocol         uint32                       `json:",omitempty"` // EX: TCP = 6, UDP = 17
+	InternalPort     uint16                       `json:",omitempty"`
+	ExternalPort     uint16                       `json:",omitempty"`
+	DistributionType LoadBalancerDistribution     `json:",omitempty"`
```

Comment:
Could we add the // EX: examples to the DistributionType like it is for protocol? Will make it clear later when someone looks at the code

Reply:
yes, the DistributionType field is also explained in the definition of the LoadBalancerDistribution enum on lines 58-66